generator/src/main/java/com/algorand/sdkutils/generators/Generator.java

```diff
@@ -317,14 +319,16 @@ static String getImports(Map<String, Set<String>> imports) {
         return sb.toString();
     }
 
+    // getPropertyWithJsonSetter formats the property into java declaration type with 
+    // the appropriate json annotation.
     static String getPropertyWithJsonSetter(TypeDef typeObj, String javaName, String jprop){
         StringBuffer buffer = new StringBuffer();
 
         if (typeObj.isOfType("getterSetter")) {
             return typeObj.def.toString();
         }
 
-        switch (typeObj.typeName) {
+        switch (typeObj.javaTypeName) {
         case "java.util.DateTime":
             throw new RuntimeException("Parsing of time is not yet implemented!");
```

Comment:
I thought we had a couple date fields?

Reply:
Yes. We have it for paths. We encode dates but not decode them from responses.